Assets/MRTK/Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -744,17 +746,25 @@ public IEnumerator TestInteractableToggleCollectionIndexUpdate()
         [UnityTest]
         public IEnumerator TestButtonStateResetWhenFocusLostAfterPinch()
         {
+            /// This test breaks if there is roll on the camera. I don't know if that's a real
```

Comment:
@vaoliva Note that I had to force the camera roll to zero to pass this test. But I don't understand the test enough to know if that's expected or not. With camera roll set to 15 degrees, the first HasFocus is false (and slowing down time with a waitforseconds confirms that the ray isn't on the box).

Any thoughts there? Thanks!